### ISSUE TITLE
NVDA no longer speaks page numbers

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#200]
**Reported by chris1988 on 21 Jun 2008 13:24 UTC**
In the document formatting prefrences, there is an option that says "speak pages".
This currently doesn't work.  In a word document, you can scroll page by page and nvda currently doesn't speak the different page numbers even with the option checked.

**Comment by nvdaTrac** [comment]
**Comment by chris1988 on 21 Jun 2008 13:25 UTC**
Oopps, i need to change to minor.
Replying to [chris1988](ticket:120):
> In the document formatting prefrences, there is an option that says "speak pages".
> This currently doesn't work.  In a word document, you can scroll page by page and nvda currently doesn't speak the different page numbers even with the option checked.

**Comment by nvdaTrac** [comment]
**Comment by bernd Dorer on 3 Aug 2008 10:51 UTC**
It even doesn't work, if you are reading a document, so you havn't a chance to get the page number with NVDA.

**Comment by nvdaTrac** [comment]
**Comment by Bernd on 5 Sep 2008 10:01 UTC**
I noticed, when you reading by Arowing with up and down, by review cursor, or by say all, NVDA reads Page numbers. But if you reading by mouse or if you are pressing page up and down NVDA won't speak page numbers.

**Comment by nvdaTrac** [comment]
**Comment by Bernd on 5 Sep 2008 10:55 UTC**
Replying to [Bernd](comment:3):
> I noticed, when you reading by Arowing with up and down, by review cursor, or by say all, NVDA reads Page numbers.
Sorry, I noticed wrong:
> But if you are pressing page up and down NVDA won't speak page numbers.

If you navigating with page up and down, NVDA will read page numbers, too. But when you navigating with Control+Page up and down by page, NVDA won't read page lines.

**Comment by nvdaTrac** [comment]
**Comment by pvagner on 11 Sep 2008 11:53 UTC**
Speaking page and line numbers has been fixed about a week before around r2357. Anouncing current caret offset after pressing Ctrl+pageup and ctrl+pagedown has been added in r2378
